source/solvers/gls_navier_stokes.cc

```diff
@@ -961,9 +1488,17 @@ GLSNavierStokesSolver<dim>::assemble_matrix_and_rhs(
     {
       if (time_stepping_method ==
           Parameters::SimulationControl::TimeSteppingMethod::bdf1)
-        assembleGLS<true,
-                    Parameters::SimulationControl::TimeSteppingMethod::bdf1,
-                    Parameters::VelocitySource::VelocitySourceType::none>();
+        { // TEMPORARY free surface implementation
```

Comment:
Is this still temporary?

Reply:
I guess no, if we keep this specialized assembler!